auacm/angular/src/app/blog-list/blog-list.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="body">
+  <!-- Container -->
+  <div class="col-md-8 col-md-offset-2">
+    <div class="page-header">
+      <h1>Auburn ACM Blog</h1>
+    </div>
+    <div class="row blog-list" (mouseover)="hoverId = post.id" (mouseleave)="hoverId = -1" *ngFor="let post of blogPosts;">
```

Comment:
Oh yeah, that probably had the index declaration inside of the ngFor loop, but was removed later on.